packages/web-frontend/src/screens/LandingPage/LandingPageFooter.js

```diff
@@ -9,6 +9,7 @@ import { Link, Typography, List } from '@material-ui/core';
 import { useCustomLandingPages } from './useCustomLandingPages';
 
 const Footer = styled.footer`
+  margin-top: auto;
```

Comment:
this keeps the footer at the bottom in the flex column before the content of the landing pages has loaded